### PR TITLE
Migrate PLG to Prod

### LINKED ISSUE #3222: PLG Production Migration
**Description:**
As a follow on to #3046 we want to move the PLG stack from the dev environment to the production environment.

**Acceptance Criteria:** 
_Create a list of functional outcomes that must be achieved to complete this issue_

- [ ] PLG is deployed in prod
- [ ] Dev and Staging apps are still observable from PLG in prod
- [ ] Testing Checklist has been run and all tests pass
- [ ] README is updated, if necessary

**Tasks:**
_Create a list of granular, specific work items that must be completed to deliver the desired outcomes of this issue_ 

- [x] Update networking commands in `plg/deploy.sh` to be based on prod instead of dev (make Grafana route private, change `-s` switch to point to prod, etc...)
- [x] Update nginx cloud conf to proxy Grafana in the same way we do locally
- [x] Update `deploy-frontend.sh` networking to correctly proxy Grafana in the same way we do with Kibana
- [x] Use the `plg/deploy.sh` script to deploy PLG AND a prod postgres-exporter to the Prod env
- [x] Use the `plg/deploy.sh` script to re-deploy the postgres-exporter for `dev` and `staging` environments
- [x] Update `deploy-backend.sh` and `deploy-fronend.sh` PLG networking to tunnel routes to prod PLG
- [x] Run Testing Checklist and confirm all tests pass

**Notes:**
_Add additional useful information, such as related issues and functionality that isn't covered by this specific issue, and other considerations that will be helpful for anyone reading this_ 

- _Note 1_
- _Note 2_
- _Note 3_

**Supporting Documentation:**
_Please include any relevant log snippets/files/screen shots_

- _Doc 1_
- _Doc 2_

**Open Questions:**
_Please include any questions or decisions that must be made before beginning work or to confidently call this issue complete_ 

- _Open Question 1_
- _Open Question 2_

### PR DESCRIPTION
**Pull request by elipe17**
## Summary of Changes
- Updated cloud.gov nginx conf to proxy Grafana
- Updated networking to correctly tunnel routes to and from the production space
- Fixed grafana config to correctly auto upload resources and set more conformant session limitations
- Removed some extremely annoying log messages
- Updated Loki to have 1GB of memory
Pull request closes #3222 

## How to Test
1. Deploy the PR to an environment of your choice. 
2. Use the CF CLI to verify that PLG is not in the dev space anymore. If you have the privilege, verify the apps are only in prod.
3. Give yourself sys admin or developer role and verify the frontend will proxy grafana in the prod env
4. Reach out to me for credentials to log into grafana since we don't have sso setup

## Deliverables
_More details on how deliverables herein are assessed included [here](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverables)._

### [Deliverable 1: Accepted Features](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-1-Accepted-Features)

Checklist of ACs:
- [x] PLG is deployed in prod
- [x] Dev and Staging apps are still observable from PLG in prod
- [ ] Testing Checklist has been run and all tests pass
- [x] README is updated, if necessary

### [Deliverable 2: Tested Code](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-2-Tested-Code)

+ Are all areas of code introduced in this PR meaningfully tested?
  + [x] If this PR introduces backend code changes, are they meaningfully tested?
  + [x] If this PR introduces frontend code changes, are they meaningfully tested?
+ Are code coverage minimums met?
  + [x] Frontend coverage: [_insert coverage %_] (see `CodeCov Report` comment in PR)
  + [x] Backend coverage: [_insert coverage %_] (see `CodeCov Report` comment in PR)

### [Deliverable 3: Properly Styled Code](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-3-Properly-Styled-Code)

+ [x] Are backend code style checks passing on CircleCI?
+ [x] Are frontend code style checks passing on CircleCI?
+ [x] Are code maintainability principles being followed?

### [Deliverable 4: Accessible](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-4-Accessibility)

+ [ ] Does this PR complete the epic? 
+ [ ] Are links included to any other gov-approved PRs associated with epic?
+ [ ] Does PR include documentation for Raft's a11y review? 
+ [ ] Did automated and manual testing with `iamjolly` and `ttran-hub` using Accessibility Insights reveal any errors introduced in this PR?

### [Deliverable 5: Deployed](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-5-Deployed)

+ [x] Was the code successfully deployed via automated CircleCI process to development on Cloud.gov?

### [Deliverable 6: Documented](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-6-Code-documentation)

+ [x] Does this PR provide background for why coding decisions were made?
+ [ ] If this PR introduces backend code, is that code easy to understand and sufficiently documented, both inline and overall?
+ [ ] If this PR introduces frontend code, is that code easy to understand and sufficiently documented, both inline and overall?
+ [ ] If this PR introduces dependencies, are their licenses documented?
+ [x] Can reviewer explain and take ownership of these elements presented in this code review?

### [Deliverable 7: Secure](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-7-Secure)

+ [x] Does the OWASP Scan pass on CircleCI?
+ [ ] Do manual code review and manual testing detect any new security issues?
+ [ ] If new issues detected, is investigation and/or remediation plan documented? 

### [Deliverable 8: User Research](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-8-User-Research)

Research product(s) clearly articulate(s):
+ [ ] the purpose of the research
+ [ ] methods used to conduct the research 
+ [ ] who participated in the research
+ [ ] what was tested and how
+ [ ] impact of research on TDP
+ [ ] (_if applicable_) final design mockups produced for TDP development

### CODE REVIEW

**Comment by lhuxraft** [comment]
[DONE] Attach to 3222

**Review by andrew-jameson — APPROVED** [review]

**Review by jtimpe — APPROVED** [review]

**Review by raftmsohani — APPROVED** [review]
LGTM!

**Comment by ADPennington** [comment]
@elipe17 cf cli issue in the pipeline affecting the deployment on this branch.

File: tdrs-backend/plg/README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
...
+
+Grafana has three roles: `Admin`, `Editor`, and `Viewer`. We have also defined two teams (groups) in Grafana: `OFA` and `TDP` and several users. The teams are how we manage least privilege to Grafana's resources. Upon creation, all users are given one of the base roles. All TDP dev user accounts are given read only access (`Viewer`) to Grafana and OFA has a user account(s) associated with each of the roles. All users who are outside of OFA should always be assigned the `Viewer` role to maintain least privilege. All dashboards in Grafana are viewable by team as opposed to individual users/roles. Dashboard permissions are configured per dashboard and each team is given read only access to the appropriate dashboards. The `ofa-admin` user is the only direct user given access to resources. This account is given exclusive admin rights to all of Grafana.
```
Comment (ADPennington) [inline tdrs-backend/plg/README.md]: @elipe17 please confirm if "TDP"is a team name. I didn't see this.

**Review by ADPennington — APPROVED** [review]
@elipe17 this LGTM! testing notes below and one question. 

---

- Confirmed ability to get to grafana with sys admin permissions
- Confirmed inability to get to grafana w/o sys admin permissions
- Confirmed PLG stack not available in `tanf-dev` space (via `cf apps`)
- Confirmed PLG apps only available in `tanf-prod` space 
- Confirmed the following least privilege protocols over access to grafana are in-place:
  - Access to grafana requires to levels of sign-in. First, sign-in via TDP authentication service, then approved users with developer or sys admin permissions can access grafana endpoint. Second, must sign-in with grafana-specific user/pass credentials. 
  - admin/ofa users have more visibility than non-admin users
  - non-admin user has visibility to dev and staging logs

**Pull request merged**